client/src/App.js

```diff
@@ -14,12 +14,14 @@ import NewQuestion from "./Components/NewQuestion.js";
 
 import Questions from "../src/mockData/Questions.json";
 export function App() {
-
-	// const [message, setMessage] = useState("Loading...");
-
-	// useEffect(() => {
-	// 	getMessage().then((message) => setMessage(message));
-	// }, []);
+	const [questionData, setQuestionData]=useState({});
+	const [route, setRoute]=useState("quizes");
```

Comment:
```suggestion
	const [route, setRoute]=useState("quizzes");
```

Reply:
Thanks Shanthi, it was my mistake at the beginning used quizes but I'm going to fix it.